litmus-portal/README.md

```diff
@@ -40,7 +40,7 @@ Or
 export LITMUS_PORTAL_NAMESPACE="<namespace>"
 kubectl create ns ${LITMUS_PORTAL_NAMESPACE}
 kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/litmus-portal-crds.yml
```

Comment:
Can we update this as well @Jonsy13 ?

Reply:
For now, we are maintaining CRDs in master only.